docs/README.md

```diff
@@ -1,60 +1,46 @@
 
 # Shelf
 
-The Shelf is a theme block responsible for showing a **collection of products** in the home page. 
+The Shelf is a theme block responsible for showing a **collection of products** in the home page.
 
 ![shelf](https://user-images.githubusercontent.com/52087100/70079904-60dc5280-15e4-11ea-8ef6-0aa69cadd61d.png)
 
 ## Configuration
 
-1. Import the Shelf app to your theme's dependencies on the `manifest.json`, for example:
+Our new approach to the Shelf is through a *list of products* using the `list-context.product-list` block, a `product-summary.shelf` and a `slider-layout`.
+
+1. Add the `ProductSummary` app to your theme's dependencies on the `manifest.json`, for example:
 
 ```json
-  dependencies: {
-    "vtex.shelf": "1.x"
+  "dependencies": {
+    "vtex.product-summary": "2.x"
   }
 ```
 
-2. Add the `shelf` block into your theme. The Shelf block queries a list of products and it can be added into any template of your theme. For it to properly function, the [**Product Summary**](https://github.com/vtex-apps/product-summary) also needs to be added to the Shelf. Check an implementation example below:
-
+2. Add the `list-context.product-list`, `product-summary.shelf` and `slider-layout` blocks into your theme. These block comes from [**Product Summary**](https://github.com/vtex-apps/product-summary) and [**Slider-Layout**](https://github.com/vtex-apps/slider-layout). Check an implementation example below:
 
 ```json
 {
-  "shelf": {
-    "blocks": ["product-summary.shelf"],
-    "props": {
-      "category": 1,
-      "orderBy": "OrderByTopSaleDESC",
-      "paginationDotsVisibility": "desktopOnly",
-      "productList": {
-        "maxItems": 8,
-        "itemsPerPage": 4,
-        "minItemsPerPage": 1,
-        "scroll": "BY_PAGE",
-        "arrows": true,
-        "titleText": "Top sellers"
-      }
-    }
+  "product-summary.shelf#demo1": {
+    "children": [
+      "stack-layout#prodsum",
+      "product-summary-name",
+      "product-rating-inline",
+      "product-summary-space",
+      "product-summary-price",
+      "product-summary-buy-button"
+    ]
   },
-  
-  "product-summary": {
-  "props": {
-    "isOneClickBuy": false,
-    "showBadge": true,
-    "badgeText": "OFF",
-    "displayBuyButton": "displayButtonHover",
-    "showCollections": false,
-    "showListPrice": true,
-    "showLabels": false,
-    "showInstallments": true,
-    "showSavings": true
+  "list-context.product-list#demo1": {
+    "blocks": ["product-summary.shelf#demo1"],
+    "children": ["slider-layout#demo-products"]
   }
 }
-
-}
 ```
 
+The `list-context.product-list` is the block responsible for performing the GraphQL query that fetches the list of products and its props can be found below.
+
 :warning: `RelatedProducts` is a subtype of a Shelf block (`shelf.relatedProduct`) that queries and displays the related products on a Product Details Page. It can therefore only be declared in a product template (`store.product`), for example:
```

Comment:
## Related Products Shelf 

`RelatedProducts` is a subtype of a Shelf block (`shelf.relatedProduct`) that queries and displays the related products on a Product Details Page. It can therefore only be declared in a product template (`store.product`), for example: